acceptance_tests/features/sign_in_internal.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Internal user signs in
+  As an internal user
+  I need to sign in to the SDC system
+  So that I can access my SDC account and carry out a number of actions
+
+  Background: Internal user is already signed in
```

Comment:
Do we need the background here? The user is signing in in the tests so doesn't need to sign in beforehand I'd think